### PR TITLE
fix rst code block format

### PR DESCRIPTION
**Pull request by Lee-W**
Some of the code blocks in the document use markdown format, which will be rendered like the following 

圖片

https://github.com/apache/airflow/blob/main/docs/apache-airflow/howto/docker-compose/index.rst

---
**^ Add meaningful description above**
Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

File: docs/apache-airflow/security/webserver.rst [hunk 1]
```diff
@@ -313,9 +313,9 @@ setting the ``RATELIMIT_*`` configuration settings in ``webserver_config.py``.
 For example, to use Redis as a rate limit storage you can use the following configuration (you need
 to set ``redis_host`` to your Redis instance)
 
-```
-RATELIMIT_STORAGE_URI = 'redis://redis_host:6379/0
-```
+.. code-block:: ini
+
+    RATELIMIT_STORAGE_URI = 'redis://redis_host:6379/0
```
Comment (uranusjr) [inline docs/apache-airflow/security/webserver.rst]: Reading the paragraph above, I _think_ this should be Python code instead?

File: docs/docker-stack/entrypoint.rst [hunk 2]
```diff
@@ -43,9 +43,9 @@ This can be achieved in various ways - you can change USER when you extend or cu
 you can dynamically pass the user to  ``docker run`` command, by adding ``--user`` flag in one of
 those formats (See `Docker Run reference <https://docs.docker.com/engine/reference/run/#user>`_ for details):
 
-```
-[ user | user:group | uid | uid:gid | user:gid | uid:group ]
-```
+.. code-block:: bash
+
+    [ user | user:group | uid | uid:gid | user:gid | uid:group ]
```
Comment (uranusjr) [inline docs/docker-stack/entrypoint.rst]: And I don’t even know what this block is intended to be…

File: docs/apache-airflow/security/webserver.rst (hunk 1 above)
Comment (Lee-W) [reply docs/apache-airflow/security/webserver.rst]: yep, this should be python instead

File: docs/docker-stack/entrypoint.rst (hunk 2 above)
Comment (Lee-W) [reply docs/docker-stack/entrypoint.rst]: hmmm.. I would say it's more "like" bash? 
https://docs.docker.com/engine/reference/run/#user

File: docs/apache-airflow/security/webserver.rst (hunk 1 above)
Comment (Lee-W) [reply docs/apache-airflow/security/webserver.rst]: just fixed it. Thanks!

File: docs/docker-stack/entrypoint.rst (hunk 2 above)
Comment (uranusjr) [reply docs/docker-stack/entrypoint.rst]: Maybe just leave this unhighlighted? (using `text`)

Comment (Lee-W) [reply docs/docker-stack/entrypoint.rst]: Sounds good. just updated

**Review by uranusjr — APPROVED** [review]

**Review by pankajastro — APPROVED** [review]

**Pull request merged**